.github/workflows/create-tag/action.yml

```diff
@@ -0,0 +1,21 @@
+name: Create Git Tag
+description: Generate a new tag and push it to the remote repository.
+
+inputs:
+  MAINTAIN_USERNAME:
+    required: true
+  MAINTAIN_EMAIL:
+    required: true
+  PACKAGE_VERSION:
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - run: |
+            git config user.name ${{ inputs.MAINTAIN_USERNAME }}
+            git config user.email ${{ inputs.MAINTAIN_EMAIL }}
+            echo "Create tag v${PACKAGE_VERSION}"
+            git tag "v${PACKAGE_VERSION}"
+            git push origin "v${PACKAGE_VERSION}"
+      shell: bash
```

Comment:
Move before the run

Reply:
If the "shell: bash" will be before the "run" the action will not work.
As the docs also mentioned, I'm getting also warning on my IDE.

https://docs.github.com/en/actions/creating-actions/creating-a-composite-action